components/sync15/src/request.rs

```diff
@@ -142,13 +151,14 @@ pub struct UploadResult {
 
 pub type PostResponse = Sync15ClientResponse<UploadResult>;
 
+//REMOVE BatchState Unnsupported for issue 1144
 #[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub enum BatchState {
-    Unsupported,
     NoBatch,
     InBatch(String),
 }
 
+// The struct holding all the options for the queue
```

Comment:
Here and elsewhere, "the struct" is redundant. Also, if the comment only contains the information that is in the name of the type, then it isn't a useful comment.

Also a `PostQueue` has rich behavior, this isn't just options--this is the struct that does the `add`ing of records and `flush`ing of batches of records to the service, so the comment is actually misleading.

This should be documented though, but please be more descriptive of _what_ it does.